modules/mongodb/src/test/java/org/testcontainers/containers/MongoDBContainerTest.java

```diff
@@ -9,22 +9,27 @@
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.TransactionBody;
+import org.assertj.core.api.Assertions;
 import org.bson.Document;
 import org.junit.Test;
 import org.testcontainers.utility.DockerImageName;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Let's keep the static import instead.

Reply:
I don't how/why this change got in there.  I prefer the static imports so this change is the opposite of what I'd consciously choose.  I wonder if IDEA "helped" me out here a little.